src/main/steps/respondent/respondentcaseSequence.ts

```diff
@@ -1,3 +1,6 @@
+import { YesOrNo } from 'app/case/definition';
+import { Sections, Step } from '../constants';
+import { DETAILS_KNOWN, RESPONDENT_TASK_LIST_URL, START_ALTERNATIVE, MIAM_START, MIAM_ATTEND_WILLINGNESS, PRIVATE_DETAILS_CONFIRMED, PRIVATE_DETAILS_NOT_CONFIRMED, MIAM_SUMMARY, CHECK_ANSWERS, SAFETY_MAIN_PAGE, YOUR_SAFETY, DOMESTIC_ABUSE_RISK, DOMESTIC_ABUSE_RISK_NO } from '../urls';
```

Comment:
Duplicate imports, causing lint errors

Reply:
this happened a few mins ago, while merging the changes from base branch. I am resolving it